OWNERS

```diff
@@ -4,7 +4,6 @@ approvers:
   - jasondellaluce
   - fededp
   - andreagit97
-  - incertum
```

Comment:
As I step into my next professional chapter, I am officially transitioning out of my role as Falco maintainer. This decision has not been taken lightly, and while it's a farewell for now, I want to express my gratitude to this incredible team and community. Your dedication, collaboration, and passion have made this an unforgettable experience!

I'm thankful for everything we've built together, and I have no doubt the Falco community will continue to thrive. While I'm stepping away, I'll always be rooting for what comes next. Stay in touch :heart: !


```
   .:::.   .:::.
  :::::::.:::::::
  :::H4CK3R:::::  
   '::::::::::'
     '::::::'
       '::'
```

Ciao mΞl155@

Reply:
Noooooooo ☹️
I concur with @FedeDP and @leogr, you'll be missed.
Thank you so much for all the nice collaborations, reviews, etc. (and getting me on board) ! 🔧🔩🪚
Good luck in the next chapter of yours 🚀
(approve is ready to revert this patch 🙂)

Reply:
Sad to hear that, Melissa! ☹️ 
It was a pleasure working with you on the project! Hope we will meet again on another adventure... I wish you all the best for your next chapter, I'm sure you will crush it :rocket: :heart: